maths/euclidean_distance.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+def euclidean_distance(v1, v2) -> double:
+    """
+    Calculate the distance between the two = endpoints of two vectors, v1 and v2.
+    >>> euclidean_distance(np.array([0, 0]), np.array([2, 2]))
+    2.8284271247461903
+    >>> euclidean_distance(np.array([0, 0, 0]), np.array([2, 2, 2]))
+    3.4641016151377544
+    >>> euclidean_distance(np.array([1, 2, 3, 4]), np.array([5, 6, 7, 8]))
+    8.0
```

Comment:
You can test if your implementation is correct by using an actual library function.
```euclidean_distance(v1,v2) == numpy.linalg.norm(v1-v2)```
I use this trick to check if my implementation of some function is equivalent to an actual tried and tested library function. If you do so, you will need to use a flooring function since it is impossible to match every point after the decimal.